### DISCUSSION TITLE
pufff

### DISCUSSION THREAD

**Discussion by Newbytee** [darltrash/metahome#3] [blablabla]
pufffffff

icinsluvely not working with gl 3

puf i u se mesa 2 and only opengl 1 is implemnted...........

puffffff no template

**Comment by darltrash** [comment]
@Newbytee thanks man, this has been a truly revealing experience 👍

**Comment by mckaygerhard** [comment]
undertale 7

**Comment by mckaygerhard** [comment]
you two guys are just envy ok.. ! and ok is not Ogl supoprted cos i pretty young software

**Reply by darltrash** [reply]
Nah i didnt support it YET because sokol does not support it, but there are plans to do so

**Comment by Miqueas** [comment]
__i nEED tHIS pROBLEM fIXED rIGHT nOW__

**Comment by darltrash** [comment]
it works with gles2 and glcore3 now :)

**Comment by mckaygerhard** [comment]
now finally it works in the hardware of the people that really find interesting to play this game.. XD
